### PR TITLE
[tsh] Initial support for teleport tsh

### PR DESCRIPTION
**Pull request by Nuru**
## what
Initial support for [Teleport](https://gravitational.com/teleport)
## why
Customer request plus Teleport support for Geodeisc

### CODE REVIEW

File: rootfs/templates/wrapper [hunk 1]
```diff
@@ -150,7 +150,7 @@ function use() {
 		--publish ${GEODESIC_PORT}:${GEODESIC_PORT}
 		--name "${DOCKER_NAME}"
 		--rm
-		--env KUBERNETES_API_PORT=${GEODESIC_PORT})
+		--env GEODESIC_PORT=${GEODESIC_PORT})
```
Comment (goruha) [inline rootfs/templates/wrapper]: I'm not sure about ports

Comment (goruha) [reply rootfs/templates/wrapper]: @osterman

Comment (osterman) [reply rootfs/templates/wrapper]: I support this. I want to generalize it so we can use it for both `kubectl` and `teleport`. The port can be overloaded,

Comment (osterman) [reply rootfs/templates/wrapper]: Also, I like the consistency where the LHS==RHS env name

**Pull request merged**